assets/js/components/setup/SetupUsingProxyWithSignIn.js

```diff
@@ -95,6 +99,24 @@ export default function SetupUsingProxyWithSignIn() {
 		};
 	} );
 
+	const dashboardURL = useSelect( ( select ) =>
+		select( CORE_SITE ).getAdminURL( 'googlesitekit-dashboard' )
+	);
+
+	const canViewSharedDashboard = useSelect( ( select ) =>
+		select( CORE_USER ).hasCapability(
+			'googlesitekit_view_shared_dashboard'
```

Comment:
I think this capability should be added to `assets/js/googlesitekit/datastore/user/constants.js` as `PERMISSION_VIEW_SHARED_DASHBOARD`.

Reply:
I was wondering about that and was surprised it hadn't been. Will-do 👍🏻